frontend/src/pages/Dashboard.jsx

```diff
@@ -47,7 +47,7 @@ class Dashboard extends Component {
         }
 
         //Gets rid of the search in window.location
-        var myNewURL = window.location.protocol + '//' + window.location.hostname + window.location.pathname;
+        var myNewURL = window.location.protocol + '//' + window.location.hostname + ':' + window.location.port + window.location.pathname;
```

Comment:
was this tested with 80 as a port? how does the url look like? will the port be sent? e.g. `spoti-vote.com:80/dashboard` ?

Reply:
no idee my port 80 was taken :(

Reply:
nice